File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/PatchTriggerWorkflowExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class PatchTriggerWorkflowExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        var flagTriggerInput = new FlagTriggerInput(
            comment: "optional comment",
            instructions: JsonSerializer.Deserialize<List<Dictionary<string, object>>>("""
                [
                    {
                        "kind": "disableTrigger"
                    }
                ]
            """)
        );

        try
        {
            var response = new FlagTriggersApi(config).PatchTriggerWorkflow(
                projectKey: "projectKey_string",
                environmentKey: "environmentKey_string",
                featureFlagKey: "featureFlagKey_string",
                id: "id_string",
                flagTriggerInput: flagTriggerInput
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FlagTriggersApi#PatchTriggerWorkflow: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/PatchTriggerWorkflowExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const flagTriggerInput: api.FlagTriggerInput = {
  comment: "optional comment",
  instructions: [
    {
      "kind": "disableTrigger"
    }
  ],
};

new api.FlagTriggersApi(configuration).patchTriggerWorkflow(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "featureFlagKey_string", // featureFlagKey
  "id_string", // id
  flagTriggerInput,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling FlagTriggersApi#patchTriggerWorkflow:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
